setup_scripts/install_dependencies.sh

```diff
@@ -11,14 +11,19 @@
 # The current directory
 CURR_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 
-echo "================================================================" 
+echo "===============================================================" 
```

Comment:
I'm nitpicking a bit here, but was there something that led to the last equals sign getting removed? If not, for uniformity with line 16, it would be nice to add it back:
```suggestion
echo "================================================================" 
```

Reply:
Noted, I will add it back